internal/biz/model/common.go

```diff
@@ -334,9 +333,7 @@ type ConsoleHref string
 
 func NewConsoleHref(href string) (ConsoleHref, error) {
 	href = strings.TrimSpace(href)
-	if href == "" {
-		return ConsoleHref(""), fmt.Errorf("%w: ConsoleHref", ErrEmpty)
-	}
+	// ConsoleHref can be empty (optional field)
```

Comment:
I don't think an empty Href/URI is a valid one. It's just that we don't _have_ to have a valid one. The optionality is a rule about the entity that _has_ the href, not about the href itself.

Reply:
Yeah,you are correct. This currently violates the "Domain objects should not be initialized in an invalid state", since we are allowing a console href to be initialized with a empty value. I will move the validation into the ReporterResource entity.

Reply:
done.